src/main/java/com/mabl/integration/jenkins/MablStepDeploymentRunner.java

```diff
@@ -192,13 +193,8 @@ private boolean allPlansComplete(final ExecutionResult result) {
     }
 
     private boolean allPlansSuccess(final ExecutionResult result) {
-
-        boolean isSuccess = true;
-
-        for (ExecutionResult.ExecutionSummary summary : result.executions) {
-            isSuccess &= summary.success;
-        }
-        return isSuccess;
+        Boolean success = result.eventStatus.getSucceeded();
+        return success != null && success;
```

Comment:
nit: you can do `Boolean.TRUE.equals(success)`

Reply:
Great catch. I'll fix this.